pkg/expression/column_test.go

```diff
@@ -159,6 +162,30 @@ func TestIndexInfo2Cols(t *testing.T) {
 	require.Len(t, lengths, 2)
 	require.True(t, resCols[0].EqualColumn(col0))
 	require.True(t, resCols[1].EqualColumn(col1))
```

Comment:
Could you add a gobench in this PR? This way we can monitor the performance changes here on a daily basis.